panther_hardware_interfaces/src/panther_system.cpp

```diff
@@ -70,6 +69,15 @@ CallbackReturn PantherSystem::on_configure(const rclcpp_lifecycle::State &)
 {
   RCLCPP_INFO(logger_, "Configuring Panther System");
 
+  RCLCPP_INFO_STREAM(logger_, "Creating GPIO controller for Panther version: " << panther_version_);
+  if (panther_version_ >= 1.2 - std::numeric_limits<float>::epsilon()) {
```

Comment:
Probably better to move it into a separate method (as it is used also in other parts).